src/js/player.js

```diff
@@ -300,7 +300,17 @@ class Player extends Component {
     if (tag.parentNode) {
       tag.parentNode.insertBefore(el, tag);
     }
-    Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    //Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    // append the DOM el function to make use of our component.addChild method
+    // to properly be injected into the children Array
+    tag.el = function () {
```

Comment:
interesting. Thanks for the update @gkatsev - If there's anything I can do to try and help, I'll do what I can.